package.json

```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite.git"
   },
   "scripts": {
-    "clean:node_modules": "pnpx rimraf --glob **/node_modules",
+    "clean:node_modules": "rimraf --glob **/node_modules",
```

Comment:
This `pnpx` is here with reason, read description:
https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/697#issue-2502937642

Reply:
Okay